ddln_imaginator.proto

```diff
@@ -28,6 +28,7 @@ message ImageRequest {
     optional CameraIntrinsics intrinsics = 4;
     optional Pose extrinsics_WC = 5;
     repeated SceneObject objects = 6;
+    optional bool auxiliary = 7; // request auxiliary (segmented) image instead of regular image
```

Comment:
add an enum instead, similar to `ImageFormat`
then it will be easier to add other types in the future: depth map, for example.

Reply:
good point!